background/services/chain/serial-fallback-provider.ts

```diff
@@ -43,8 +44,8 @@ function waitAnd<T, E extends Promise<T>>(
  * ms to back off before making the next attempt.
  */
 function backedOffMs(backoffCount: number): number {
-  const backoffSlotStart = BASE_BACKOFF_MS ** backoffCount
-  const backoffSlotEnd = BASE_BACKOFF_MS ** (backoffCount + 1)
+  const backoffSlotStart = BASE_BACKOFF_MS ** Math.sqrt(backoffCount)
+  const backoffSlotEnd = BASE_BACKOFF_MS ** Math.sqrt(backoffCount + 1)
```

Comment:
This formula does not make dimensional sense (we are raising seconds to a fractional power - I hope we don't have any physicists in the team :)

I think we are looking for this instead (standard exponential backoff):

```suggestion
  const backoffSlotStart = BASE_BACKOFF_MS * 2 ** backoffCount
  const backoffSlotEnd = BASE_BACKOFF_MS * 2 ** (backoffCount + 1)
```

If not, please add a comment to the code explaining why.

Reply:
The idea here was just to get our backoff time to be a little less aggressive (right now it averages something like 150ms backoff, 19s backof, 6 min backoff - and this made it more like 150ms, 1s, 4s, 10s, etc..).  From further testing it looks like an **even more** aggressive backoff timing than what we have right now may be desirable so this looks like it's definitely going to change.